config/examples/cache_store.yml

```diff
@@ -1,14 +1,12 @@
-development:
+base: &default
   - :mem_cache_store
   - <%= ENV.fetch('MEMCACHE_SERVERS', 'localhost:11211') %>
 
+development:
+  <<: *default
+
 test:
   - :null_store
```

Comment:
I wonder why keep `null_store` for testing if we use in dev mode.

Reply:
@jlledom may have opinion

Reply:
I'm not sure about enabling the cache for tests. The reason for doing it is detecting the subtle kind of errors the cache can produce in production. But enabling it for tests could break tests in a non deterministic way depending on the tests execution order, so we should ensure the cache is reset between tests, which would make the tests slower. I don't think is worth it. I'd rather enable it for some particular tests which we know the cache is involved in some way, but not for all since it would impact performance and would not have any effect on most tests.